src/bp/ui-shared-lite/ConfirmDialog/index.tsx

```diff
@@ -0,0 +1,94 @@
+// @ts-nocheck
+import { Button, Classes, Icon, Intent } from '@blueprintjs/core'
+import '@blueprintjs/core/lib/css/blueprint.css'
+import React, { FC } from 'react'
+import ReactDOM from 'react-dom'
+
+import { Body, Footer, Wrapper } from '../Dialog'
+
+import styles from './style.scss'
+import { ConfirmDialogOptions, ConfirmDialogProps } from './typings'
+
+const ConfirmDialogComponent: FC<ConfirmDialogProps> = props => {
```

Comment:
When you move a component from `ui-shared` to `ui-shared-lite`, you should remove it from `ui-shared`, then import it from `ui-shared-lite`, then export it from `ui-shared`.

This way the code is not duplicated in both places, and you don't have to refactor places where it's already being used because it's exported neatly. You can have a look at how it's done for the components in shared lite (ex: MoreOptions)

Reply:
Thanks for the tip! I agree that it is a cleaner way of handling this. I'll make the changes!